skfda/exploratory/stats/_stats.py

```diff
@@ -99,6 +100,59 @@ def cov(
     return X.cov(ddof=ddof)
 
 
+@functools.singledispatch
+def std(X: F, ddof: int = 1) -> F:
+    r"""
+    Compute the standard deviation of all the samples in a FData object.
+
+    .. math::
+        \text{std}_X(t) = \sqrt{\frac{1}{N-\text{ddof}}
+        \sum_{n=1}^{N}{\left(X_n(t) - \overline{X}(t)\right)^2}}
+
+    Args:
+        X: Object containing all the samples whose standard deviation is
+            wanted.
+        ddof: Means "Delta Degrees of Freedom". The divisor used in
+            calculations is `N - ddof`, where `N` represents the number of
+            samples in `X`. By default ddof is 1.
+
+    Returns:
+        Standard deviation of all the samples in the original object, as a
+        :term:`functional data object` with just one sample.
+
+    """
+    raise NotImplementedError("Not implemented for this type")
+
+
+@std.register(FDataGrid)
```

Comment:
Is it really necessary to specify the type here?

Reply:
It is not. Removed.